flight/UAVObjects/uavobjectmanager.c

```diff
@@ -74,7 +74,7 @@ struct ObjectEventEntry {
 struct ObjectEventEntryThrottled {
 	struct ObjectEventEntry   entry; // MUST be first! So throttled entry can be interpreted as ObjectEventEntry
 
-	uint32_t                  due;
+	uint32_t                  when;
```

Comment:
Changed from due time to when last happened to allow use of Thread_Period_Elapsed instead of doing something clever ourselves.